firefox_ui_harness/runners/base.py

```diff
@@ -24,7 +24,7 @@
     'browser.search.update': False,
     'browser.sessionstore.resume_from_crash': False,
     'browser.shell.checkDefaultBrowser': False,
-    'browser.startup.page': 0,
+    'browser.startup.page': 3,
```

Comment:
I'm assuming this was 0 before for a reason, so I don't think we can change this here.